src/com/google/javascript/jscomp/DefaultPassConfig.java

```diff
@@ -174,6 +174,17 @@ void maybeInitializePreprocessorSymbolTable(AbstractCompiler compiler) {
       }
     }
   }
+  
+  @Override
+  protected List<PassFactory> getAlwaysPreChecks() {
+    List<PassFactory> preChecks = new ArrayList<>();
+    
+    if (options.wrapGoogModulesForWhitespaceOnly) {
```

Comment:
Actually I don't know what you mean about making this tied to closurePass. The point of whitespace only is to make it bundled with minimal changes and as fast as possible. Or at least that's my reason for using it. Options.closurePass is false in whitespace only mode.

I think you mean whitespace only mode can't work without this, so we should remove the opt-in / always enable this pass in whitespace-only mode. 

But whitespace only mode skips all non transpilation passes, and I'm not transpiling (why would I in normal development, now Chrome stable supports so many ES6 features?), so no passes are done. Hence the new "always pre checks" stage.

Incidentally why does the debugger disallow ES6 out (https://closure-compiler-debugger.appspot.com/#input0%3Dlet%2520x%2520%253D%25205%253B%250A%26input1%26conformanceConfig%26externs%26refasterjs-template%26includeDefaultExterns%3D1%26LANG_IN_IS_ES6%3D1)? I use this in Chrome 45 via the Java API as part of the lean and fast whitespace only mode (with this PR).